learn/tanssi/templates.md

```diff
@@ -1,30 +1,54 @@
+- **Customizability** - Appchain templates are a great starting point and are completely customizable. The functionalities and default configurations they include can be modified, replaced, or extended to meet the specific requirements of the use case
 
-- **Upgrade and Compatibility** - Substrate is an evolving framework, with new features, enhancements, and bug fixes being regularly introduced, and the provided Tanssi templates are kept up to date with these upgrades
+- **Upgrade and Compatibility** - Substrate is an evolving framework, with new features, enhancements, and bug fixes being regularly introduced, and the provided Tanssi Appchain templates are kept up to date with these upgrades
 
 ## Start building a ContainerChain {: #start-building } 
 
-To start building a ContainerChain to deploy in Tanssi, some useful templates to kick-start the development process are provided in the [official repository](https://github.com/moondance-labs/tanssi){target=blank}:
+To start building a ContainerChain to deploy in Tanssi, some useful Appchain templates to kick-start the development process are provided in the [official repository](https://github.com/moondance-labs/tanssi){target=blank}.
 
-- **Simple Template** - this node template is based on the [parachain Substrate template](https://github.com/substrate-developer-hub/substrate-parachain-template){target=_blank} featuring [Cumulus](https://github.com/paritytech/cumulus/){target=blank} to allow the ContainerChain to work properly in the Polkadot ecosystem. It also includes Tanssi's own pallets to support the protocol and some other basic pallets such as *pallet-balances* for currency operations
+The process is as simple as:
 
-- **Ethereum Virtual Machine** - this node template is based on [Frontier](https://github.com/paritytech/frontier){target=_blank}, a project that adds an Ethereum compatibility layer for Substrate. This template provides not only Tanssi protocol support but an EVM and full Ethereum support (exposing the expected RPC endpoints and emulating Ethereum's block production mechanism) allowing existing DApps to be deployed and run smoothly on your Substrate Appchain and seamlessly integrate and use popular tools such as [MetaMask](https://metamask.io){target=blank}, [Truffle](https://trufflesuite.com){target=blank}, [Remix](https://remix.ethereum.org){target=blank}, block explorers and many other
+1. Select one of the templates
+2. Add the specific logic to adapt the runtime to the requeriments of the use case
+3. Deploy in Tanssi
 
-With the EVM template, developers can have a [Moonbeam](https://moonbeam.network){target=blank}-like ContainerChain in no time and add their custom logic and features specific to the use case.
+![Using Templates to Speed Up the Developmet Process](/images/learn/tanssi/templates/templates-1.png)
+
+The two included templates are *Simple* and *EVM*, which are presented in the following sections.
+
+### Simple Template  {: #simple-template } 
```

Comment:
I don't really like the name "Simple Template"

Maybe we can do "Baseline Substrate Template" and "Baseline EVM Template"?

Reply:
Maybe consider putting the templates in their own section with whatever names you decide on for the templates:

- Start Building a ContainerChain
- Available Templates
    - ~~Simple~~ ? Template
    - EVM Template

Reply:
Let's go with "Baseline Appchain Template"